src/pyhf/infer/mle.py

```diff
@@ -47,6 +47,14 @@ def twice_nll(pars, data, pdf):
     return -2 * pdf.logpdf(pars, data)
 
 
+def _validate_fit_inputs(init, bounds, fixed_params):
```

Comment:
```suggestion
def _validate_fit_inputs(init_pars, par_bounds, fixed_params):
```

Could match the argument names established already for consistency